isign/codesig.py

```diff
@@ -220,10 +219,17 @@ def update_offsets(self):
         self.construct.length = len(superblob) + 8
         self.construct.bytes = superblob
 
-    def resign(self, app, signer):
-        self.set_entitlements(app.entitlements_path)
+    def resign(self, bundle, signer):
+        # this is tricky. We set entitlements only if the the bundle is an App, anyway
```

Comment:
Could link to https://github.com/saucelabs/isign/blob/master/docs/codedirectory.rst here